xarray/plot/animate.py

```diff
@@ -0,0 +1,179 @@
+"""
+Use this module directly:
+    import xarray.animate as xanim
+
+Or use the methods on a DataArray:
+    DataArray.plot.animate_____
+
+Or supply an ``animate_over`` keyword
+argument to a normal plotting function:
+    DataArray.plot._____(animate_over='__')
+"""
+
+import datetime
+
+import numpy as np
+import pandas as pd
+
+from .plot import _infer_line_data
+from .utils import (_ensure_plottable, _interval_to_mid_points, _update_axes,
+                    _valid_other_type, get_axis, _rotate_date_xlabels,
+                    _check_animate_over, _transpose_before_animation)
+
+
+def animate_line(darray, animate_over=None, **kwargs):
```

Comment:
Weak preference for `animate` instead of  `animate_over` i.e.
`da.plot.line(x='lat', animate='time')`